### DISCUSSION TITLE
Flutter test passing on local but failing in codemagic

### DISCUSSION THREAD

**Discussion by markjoseultra** [codemagic-ci-cd/codemagic-docs#2217] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Clone this repo https://github.com/markjoseultra/code_magic_failing_test_reproduction
2. run test on code magic

### Expected results

The test on local is passing it should also be passing in code magic

### Actual results

The test on local is passing but fails in code magic

### Build id (optional)

65603a703224071eb254b262

**Comment by himesh-cm** [comment]
Hi @markjoseultra, The test fails, with logs 
`_TextWidgetFinder:<Found 0 widgets with text "November 22, 2023 to December 25, 2023": []>`
For further debugging, please Ensure that the widget tree actually contains a widget with the text "November 22, 2023 to December 25, 2023" at the time the test runs.

**Comment by himesh-cm** [comment]
Hi @markjoseultra, I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message.
